clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/model/AmazonBlockDevice.groovy

```diff
@@ -75,4 +77,69 @@ class AmazonBlockDevice {
    */
   Boolean encrypted
 
+  private AmazonBlockDevice(Builder builder) {
```

Comment:
this doesn't appear to be used anywhere, can we roll the changes to this file back?

Reply:
The builder is used in `InstanceTypeUtils.java` (probably need to load the diff for this file as it's a huge diff). 

I needed the builder to make the `BlockDeviceConfig` code that was converted from Groovy to Java readable.